satellite/orders/endpoint.go

```diff
@@ -110,8 +121,19 @@ func (endpoint *Endpoint) Settlement(stream pb.Orders_SettlementServer) (err err
 
 	log := endpoint.log.Named(peer.ID.String())
 	log.Debug("Settlement")
+
+	requests := make([]*ProcessOrderRequest, 0, endpoint.settlementBatchSize)
+
+	defer func() {
+		if len(requests) > 0 {
+			err = endpoint.processOrders(ctx, stream, requests)
```

Comment:
We should use `errs.Combine` to not toss out any other error that may have happened.

Reply:
Updated